open_samus_returns_rando/custom_pickups.py

```diff
@@ -0,0 +1,30 @@
+from open_samus_returns_rando.patcher_editor import PatcherEditor
+
+
+def _get_actor(editor: PatcherEditor):
+    actor_to_copy = {
+        "scenario": "s000_surface",
+        "layer": "9",
+        "actor": "LE_PowerUP_Morphball",
+    }
+    new_actor = editor.resolve_actor_reference(actor_to_copy)
+    return new_actor
+
+
+def _create_baby_pickup(editor: PatcherEditor):
+    name_of_scenario = "s100_area10"
+    scenario_10 = editor.get_scenario(name_of_scenario)
+
+    baby_actor = _get_actor(editor)
+    editor.copy_actor(
+        name_of_scenario, (-3400.0, 11225.0, 0.0), baby_actor, "LE_Baby_Hatchling", 9
+    )
```

Comment:
This is what return the new actor (but as we don't need it anymore in this function it is not assigned to a variable)

Reply:
What do you mean we don't need it anymore?

Reply:
We don't need the return of it like
```
    baby_actor = editor.copy_actor(
        name_of_scenario, (-3400.0, 11225.0, 0.0), baby_actor, "LE_Baby_Hatchling", 9
    )
```
because we are not doing anything with `baby_actor` after it is created in this function.

Reply:
I'm still not understanding what you want me to fix since this works as it should.

Reply:
Nothing, this was a comment about what actually is the baby_actor :p